go.mod

```diff
@@ -103,7 +103,7 @@ require (
 	google.golang.org/genproto/googleapis/api v0.0.0-20240125205218-1f4bbc51befe // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240125205218-1f4bbc51befe // indirect
 	google.golang.org/grpc v1.61.0 // indirect
-	google.golang.org/protobuf v1.32.0 // indirect
+	google.golang.org/protobuf v1.33.0 // indirect
```

Comment:
Did you see https://github.com/golang/protobuf/issues/1596?

Looks like we'll also need to bump github.com/golang/protobuf to `v1.5.4`. We currently [have it to the incompatible `v1.5.3` version](https://github.com/openshift/csi-driver-manila-operator/blob/4c32027a95ef1eab9774910796709815aec0992a/go.mod#L49C29-L49C35).